src/doc/man/cargo-package.md

```diff
@@ -18,14 +18,19 @@ This command will create a distributable, compressed `.crate` file with the
 source code of the package in the current directory. The resulting file will be
 stored in the `target/package` directory. This performs the following steps:
 
-1. Load and check the current workspace, performing some basic checks.
-    - Path dependencies are not allowed unless they have a version key. Cargo
-      will ignore the path key for dependencies in published packages.
-      `dev-dependencies` do not have this restriction.
+1. Normalize the original `Cargo.toml`.
+    - The manifest is reformatted into a packaging-specific TOML layout.
+    - Workspace-inherited fields are resolved to their concrete values.
+    - Path and Git dependency specifications are removed.
+    - Development dependencies without a `version` key specified are removed.
+    - `[patch]`, `[replace]`, and `[workspace]` sections are removed.
+    - Fields not recognized by the current Cargo version are dropped.
+    - Certain paths, such as `package.readme` and `package.license-file`,
+      are rewritten if they point outside the package root.
+
```

Comment:
For me, the question of whether we should resolve this in docs isn't settled, see conversation at #15836